### ISSUE TITLE
Field values from external commands

### ISSUE THREAD

**Issue by borisershov** [nixys/nxs-data-anonymizer#15]
Add an ability to create a field values from external commands.

**Comment by borisershov** [comment]
The solution of this issue will be represented as an additional type `command` of column value:

```yaml

...

filters:
  some_table_name:
    columns:
      some_column_name:
        type: command
        value: /path/to/command/or/script.sh
        
...

```

If the `type: command` set for column, the value of the field `value` considered as path to command that will be executed every time for this field. An additional environment variables will be available within the runtime of the specified command:

`ENVVARTABLE={TABLE_NAME}`: contains a name of the filtered table
`ENVVARCOLUMN_{COLUMN_NAME}={COLUMN_VALUE}`: contains all columns and its values (before substitutions) for the current filtered row

The `stdout` of the command considered as new value for the filtered field.

**Comment by borisershov** [comment]
As a result the command's concept has following properties:
- The command's `stdout` will be used as a new value for the anonymized field
- Command must return zero exit code, otherwise nxs-data-anonymizer will falls with error (in this case `stderr` will be used as an error text)
- Environment variables with the row data are available within the command:
  - `ENVVARTABLE`: contains a name of the filtered table
  - `ENVVARCURCOLUMN`: contains the current column name 
  - `ENVVARCOLUMN_{COLUMN_NAME}`: contains values (before substitutions) for all columns for the current row

**Comment by borisershov** [comment]
Closed in [Release v1.4.0](https://github.com/nixys/nxs-data-anonymizer/releases/tag/v1.4.0)
